aws/resource_aws_redshift_cluster_test.go

```diff
@@ -126,6 +126,32 @@ func TestAccAWSRedshiftCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSRedshiftCluster_dnsName(t *testing.T) {
+	var v redshift.Cluster
+
+	ri := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
```

Comment:
FYI, we have a helper for this: `acctest.RandInt()`

Reply:
👍

Reply:
I created a bunch of PRs related to replacing all these across the codebase so you won't need to do anything: #4625

Reply:
oh ok, I ll revert it.